src/tctrack/track/track.py

```diff
@@ -367,6 +380,53 @@ def _get_filter_trajectories_inputs(self) -> list[str]:
 
         return inputs
 
+    def _export_or_read_inputs(self, command_name: str, inputs: list[str]) -> str:
+        """Reads/exports the command line inputs to a file and returns the command list.
```

Comment:
Surprised the linter didn't complain about the docstring not being on a clean line after the `"""`.